src/lib/y2network/interface_config_builder.rb

```diff
@@ -19,6 +19,8 @@
 require "yast"
 
 require "y2network/hwinfo"
+require "y2firewall/firewalld"
+require "y2firewall/firewalld/interface"
```

Comment:
Maybe some methods defined in https://github.com/yast/yast-yast2/blob/master/library/network/src/lib/y2firewall/helpers/interfaces.rb could be useful.

Reply:
I see that, but I worry that this class is already quite big and if I include mixin, it will be even better.

Reply:
I use from it at least that installed.